packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/audit-logs.module.js

```diff
@@ -0,0 +1,29 @@
+import angular from 'angular';
+import '@ovh-ux/ng-tail-logs';
+import '@ovh-ux/ui-kit';
+import '@uirouter/angularjs';
+import 'angular-translate';
+
+import auditLogsComponent from './audit-logs.component';
+import routing from './audit-logs.routing';
+
+const moduleName = 'ovhManagerPciProjectKubernetesDetailsAuditLogs';
+
+angular
+  .module(moduleName, [
+    'ngTailLogs',
+    'ngTranslateAsyncLoader',
+    'oui',
+    'pascalprecht.translate',
+    'ui.router',
+  ])
+  .config(routing)
+  .filter('formatDate', function() {
```

Comment:
IMO Filter name is a bit too generic.

Out of curiosity, did you know why this value isn't already formatted and accessible in the API payload?

Reply:
@antleblanc, i re-discussed with them, they said we can remove the date format from our side.
They will impalement it later.

I removed the filter and usage.
Thanks.